packages/webpack-config-terra/package.json

```diff
@@ -28,10 +28,35 @@
   },
   "homepage": "https://github.com/cerner/terra-toolkit",
   "scripts": {
-    "lint": "eslint --ext .js,.jsx ."
+    "lint": "eslint --ext .js,.jsx --ignore-path ../../.eslintignore .",
+    "jest": "jest --config ../../jest.config.js"
   },
   "dependencies": {
+    "@cerner/duplicate-package-checker-webpack-plugin": "^1.0.0",
```

Comment:
We'll move this over to terra-toolkit eventually I'm assuming once someone has extra time?

Reply:
Yep